demo/text-field-validators-demos.html

```diff
@@ -117,14 +156,13 @@ <h3>Asynchronous Custom Validator</h3>
             }
           });
         </script>
-
       </template>
     </vaadin-demo-snippet>
 
-  <dom-module id="my-ajax">
-    <template>
-    </template>
-    <script>
+    <dom-module id="my-ajax">
```

Comment:
This element doesn't seem to be used anywhere anymore so you should remove this whole `<dom-module id="my-ajax">`.